stm32f7.ld

```diff
@@ -36,7 +36,7 @@
 /* Define memory regions. */
 MEMORY
 {
-	rom (rx)  : ORIGIN = 0x08000000, LENGTH = 16K
```

Comment:
This will break all existing F7 targets in PX4. How is this supposed to work?

Reply:
Application linker scripts are already at 0x08008000  (0x08000000 + 32K = 0x08008000)
https://github.com/PX4/PX4-Autopilot/blob/master/boards/px4/fmu-v5x/nuttx-config/scripts/script.ld#L74
![image](https://user-images.githubusercontent.com/10188706/106049890-f3905880-60e6-11eb-9751-4d92795a6cfc.png)

Reply:
But what is the explanation for that change? What is being stored in the adjacent 16K? Flash based parameters? @davids5 ? Or was it really just a buffer that we added back in the day?

Reply:
With monocypher, bootloader can't fit with in 16K, and F7 is working in single-bank mode where 1 flash sector is 32K.